compiler/AST/expr.cpp

```diff
@@ -5858,124 +5885,112 @@ GenRet CallExpr::codegenPrimMove() {
   return ret;
 }
 
-bool CallExpr::codegenPrimMoveRhsIsSpecialPrimop() {
-  CallExpr* rhs    = toCallExpr(get(2));
+static bool codegenIsSpecialPrimitive(BaseAST* target, Expr* e, GenRet& ret) {
   bool      retval = false;
+  CallExpr* ce = toCallExpr(e);
```

Comment:
I agree that `ce` doesn't tell much and I can change it. But `rhs` is confusing to me. Former version of this function was called on a PRIM_MOVE CallExpr, to see if right-hand side needs special treatment. This version takes that primitive itself as an argument, so there is no right-hand side in this context.